inference-engine/src/vpu/graph_transformer/include/vpu/model/stage.hpp

```diff
@@ -144,7 +144,6 @@ VPU_DECLARE_ENUM(StageType,
     ScatterUpdate = 103,
     ReduceMin = 105,
     ExpDetectionOutput = 106,  // ExperimentalDetectronDetectionOutput
-    NonMaxSuppression = 107,
```

Comment:
Are you going to remove it from firmware?

Reply:
Yes, it will be removed separately as a part of another task - StaticShapeNMS kernel optimization.